src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Models/Apprentice/DetailsViewModel.cs

```diff
@@ -21,5 +21,6 @@ public class DetailsViewModel : IAuthorizationContextModel
         public DateTime EndDate { get; set; }
         public string ProviderRef { get; set; }
         public decimal Cost { get; set; }
+        public bool ChangeOfEmployerEnabled { get; set; }
```

Comment:
Minor, but I prefer boolean flags like this to be prefixed with _Is_ or _Can_ or similar to make the boolean nature clear in the markup.

Reply:
Changed to:
`
public bool IsChangeOfEmployerEnabled { get; set; }
`